### ISSUE TITLE
f () (( x > 0 )) and SC2210 

### ISSUE THREAD

**Issue by izabera** [koalaman/shellcheck#1150]
```sh
#!/bin/bash
f () (( x > 0 ))
```
this shouldn't trigger SC2210

**Comment by koalaman** [comment]
Totally agree. However, as a current design decision, ShellCheck only accepts `{ .. }` and `( .. )` for function definitions. This is to improve error diagnostics when people use `()` in strange places and unintentionally start a function definition.
